src/config.js

```diff
@@ -12,6 +12,8 @@ const AppConfiguration = {
   pageLimit: 20,
   schedulerFixedDelayInSeconds: 60,
   notificationSnoozeDurationInMinutes: 10,
+  defaultLanguage: 'en',
```

Comment:
default language is defined in `_/intl` -  we can use it directly (no need to create prop here) i.e check https://github.com/oVirt/ovirt-web-ui/pull/1415